tests/helpers.nim

```diff
@@ -116,4 +116,4 @@ proc checkExpiringInternal(cond: proc(): bool {.raises: [Defect], gcsafe.} ): Fu
       await sleepAsync(1.millis)
 
 template checkExpiring*(code: untyped): untyped =
-  check await checkExpiringInternal(proc(): bool = code)
+  check await checkExpiringInternal(proc(): bool = code)
```

Comment:
Your IDE seems to be configured to remove newlines at the end of files, could you make it have newlines instead? Just creates noise here

Reply:
I think I deleted it by accident